driver/pelican/main.go

```diff
@@ -151,9 +162,31 @@ func main() {
 					}
 					currentIface.PublishSignal("info", po)
 				}
+
 				time.Sleep(pollInt)
 			}
 		}()
+
+		go func() {
+			for {
+				drStatus, drErr := currentPelican.TrackDREvent()
+				if drErr != nil {
+					fmt.Printf("Failed to retrieve Pelican's DR status: %v\n", drErr)
+					done <- true
```

Comment:
A problem I noticed when testing is that if we throw an error when parsing the DR event, then the driver exits. This means that we won't be able to run Pelican drivers unless there's a DR event. Maybe we just log the error here and don't cause the DR event to crash the driver? What do folks think?

Reply:
Yes that should work. Just make sure that the driver doesn't then try to publish an empty object. Maybe do the print, then sleep for the poll duration, and then `continue`

Reply:
Or alternatively set the drStatus to `nil`